packages/manager/apps/dedicated/client/app/ip/ip/ip-ip.html

```diff
@@ -1,247 +1,179 @@
+                <p data-ng-if="alerts.antihack.length">
+                    <span
+                        class="d-block"
+                        data-ng-pluralize
+                        data-count="alerts.antihack.length"
+                        data-when="{
+                                '0': ('ip_alerts_antihack' | translate),
+                                'one': ('ip_alerts_antihack_1' | translate),
+                                'other': ('ip_alerts_antihack_other' | translate: { count: {} })
+                            }"
+                    >
+                    </span>
+                    <button
+                        class="btn btn-link"
+                        type="button"
+                        data-ng-if="!singleService()"
+                        data-ng-repeat="ip in alerts.antihack"
+                        data-ng-click="navigateToService(ip.ip.routedTo.serviceName)"
+                        data-ng-bind="ip.ip.routedTo.serviceName"
+                    ></button>
+                </p>
 
-                <h4 data-translate="ip_addresses_grid_header"></h4>
+                <p data-ng-if="alerts.arp.length">
+                    <span
+                        class="d-block"
+                        data-ng-pluralize
+                        data-count="alerts.arp.length"
+                        data-when="{
+                                '0': ('ip_alerts_arp' | translate),
+                                'one': ('ip_alerts_arp_1' | translate),
+                                'other': ('ip_alerts_arp_other' | translate: { count: {} })
+                            }"
+                    >
+                    </span>
+                    <button
+                        class="btn btn-link"
+                        type="button"
+                        data-ng-if="!singleService()"
+                        data-ng-repeat="ip in alerts.arp"
+                        data-ng-click="navigateToService(ip.ip.routedTo.serviceName)"
+                        data-ng-bind="ip.ip.routedTo.serviceName"
+                    ></button>
+                </p>
 
-                <div class="form-group w-25" data-ng-if="serviceTypes.length">
-                    <label
-                        class="control-label"
-                        for="serviceTypeSelector"
-                        data-translate="ip_table_header_service"
+                <p data-ng-if="alerts.mitigation.length">
+                    <span
+                        class="d-block"
+                        data-ng-pluralize
+                        data-count="alerts.mitigation.length"
+                        data-when="{
+                                '0': ('ip_alerts_mitigation' | translate),
+                                'one': ('ip_alerts_mitigation_1' | translate),
+                                'other': ('ip_alerts_mitigation_other' | translate: { count: {} })
+                            }"
                     >
```

Comment:
**nitpick:** Can we replace `<span>` + `.d-block` with the block level elements?